tldw_Server_API/app/api/v1/endpoints/admin/admin_sessions_mfa.py

```
@@ -86,6 +89,30 @@ async def admin_get_user_mfa_status(
+async def admin_get_bulk_mfa_status(
+    ids: str = Query(..., description="Comma-separated user IDs"),
+    principal: AuthPrincipal = Depends(get_auth_principal),
+) -> BulkMfaStatusResponse:
+    """Fetch MFA status for multiple users in a single request."""
+    user_ids = [int(uid.strip()) for uid in ids.split(",") if uid.strip().isdigit()][:200]
+
+    async def _fetch_one(uid: int) -> tuple[str, bool]:
+        try:
+            status = await admin_sessions_mfa_service.get_user_mfa_status(principal, uid)
+            return str(uid), bool(status.get("enabled", False)) if isinstance(status, dict) else False
+        except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)